IonDotnet.Tests/Internals/BinaryReaderTest.cs

```diff
@@ -57,6 +57,17 @@ public void SingleNumber(byte[] data, long value)
             ReaderTestCommon.SingleNumber(reader, value);
         }
 
+        [TestMethod]
+        [DataRow(new byte[] { 0xE0, 0x01, 0x00, 0xEA, 0x11 })]
```

Comment:
Im not sure this needs to be a parameter

Reply:
Removed parameter